### DISCUSSION TITLE
Docker set_disk_free_limit as environment variable

### DISCUSSION THREAD

**Discussion by JefSeys** [rabbitmq/rabbitmq-server#12533] [Other] [answered]
### Describe the bug

I'm trying to solve an issue and I think it is related to the disk free limit that is default set to 50MB. When trying to increase it with 'disk_free_limit.absolute', 'disk_free_limit.relative', 'RABBITMQ_DISK_FREE_ABSOLUTE_LIMIT', or 'RABBITMQ_DISK_FREE_RELATIVE_LIMIT' it doesn't do anything. I've read here (https://stackoverflow.com/questions/57262128/using-rabbitmq-with-docker-in-production) that there might not be a variable when using docker to manage this and that the solution is to change the config file. Is there anyway of using an environment variable to manage this?

### Reproduction steps

1.Make docker compose file with environment variables described above (not all at once as this will cause issues, mentioned by the docs)
2.Run and see if disk free limit is changed

### Expected behavior

When setting environment variables ('disk_free_limit.absolute', 'disk_free_limit.relative', 'RABBITMQ_DISK_FREE_ABSOLUTE_LIMIT', or 'RABBITMQ_DISK_FREE_RELATIVE_LIMIT'), I would expect to be able to manage the disk free limit.

### Additional context

Example code of docker compose:
`
services:
  rabbitmq:
    container_name: rabbitmq2222
    image: rabbitmq:4.0.2-management
    ports:
      - 15671:15671
      - 15672:15672
      - 15691:15691
      - 15692:15692
      - 25672:25672
      - 4369:4369
      - 5671:5671
      - 5672:5672
    environment:
      - disk_free_limit.absolute = 1000000000
`

**Comment by michaelklishin** [comment, marked as answer]
@JefSeys perhaps ask the maintainers of the image in https://github.com/docker-library/rabbitmq.
